src/main/java/wolox/training/models/Book.java

```diff
@@ -124,63 +126,70 @@ public String getAuthor() {
     }
 
     public void setAuthor(String author) {
-        this.author = author;
+        this.author = Preconditions.checkNotNull(author,"Author must not be null");
     }
 
     public String getImage() {
         return image;
     }
 
     public void setImage(String image) {
-        this.image = image;
+
+        this.image = Preconditions.checkNotNull(image, "Image must not be null");
```

Comment:
You are checking that the variable is not null, but you are not checking that the variable is a string other than empty (""), for this you have the following precondition:
Preconditions.checkArgument (! Variable.isEmpty (), error);
for example: Preconditions.checkArgument (! author.isEmpty (), ErrorConstants.NOT_EMPTY_AUTHOR);